golem-cluster.mini.yaml

```
@@ -0,0 +1,55 @@
+# Ray on Golem cluster name
+cluster_name: golem-cluster
+
+# The maximum number of workers the cluster will have at any given time
+max_workers: 10
+
+# The cloud provider-specific configuration properties.
+provider:
+  type: "external"
+  use_internal_ips: true
+  module: "ray_on_golem.provider.node_provider.GolemNodeProvider"
+  parameters:
+    node_config:
+      # Parameters for golem demands (same for head and workers)
+      demand:
+        image_tag: "lucjandudek/ray-on-golem:0.3.0-py3.10.13-ray2.7.1"
```

```suggestion
        image_tag: "golem/ray-on-golem:0.3.0-py3.10.13-ray2.7.1"
```
shouldn't it sport the official image? like the non-mini yaml?